src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -312,7 +314,9 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
         var user = userRepository.getUserWithGroupsAndAuthorities();
         authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.EDITOR, importedExercise, user);
         authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.EDITOR, originalModelingExercise, user);
-        exerciseService.validateScoreSettings(importedExercise);
+        // validates generall settings: points, dates
+
```

Comment:
I think you left an extra blank line here?^^

Btw. Maybe you are not yet aware of a nice command that can fix many styling issues.
Just type `yarn lint --fix` in your terminal.
 (Intellij, or cmd (make sure you are in the correct folder, e.g. C:\Users\userName\IdeaProjects\Artemis>)

Reply:
Thanks! did not know this :)